hns/hns.go

```diff
@@ -169,7 +169,7 @@ func GetHNSEndpointByName(name string) (*hcsshim.HNSEndpoint, error) {
 }
 
 func ListHNSEndpoints() ([]hcsshim.HNSEndpoint, error) {
-	endpoints, err := hcsshim.HNSListEndpointRequest("GET", "", "")
+	endpoints, err := hcsshim.HNSListEndpointRequest()
```

Comment:
Totally had this feature implemented for them :'( https://github.com/Microsoft/hcsshim/pull/104